src/pkg/build/builder_remote.go

```diff
@@ -99,6 +99,11 @@ func (rb *RemoteBuilder) Build(ctx context.Context) (err error) {
 		return err
 	}
 
+	fmt.Printf("\t+-> Build job submited with ID:\t%v\n", rd.ID.Hex())
```

Comment:
I'm not sure this `Printf` is necessary, it doesn't add anything extra since you're giving instructions below on how to get to the image.